### DISCUSSION TITLE
什么时候更新？

### DISCUSSION THREAD

**Discussion by yufulings** [monlor/docker-xiaoya#241] [Q&A]
Alist很久没更新了

**Comment by Victor2333** [comment]
自己fork
自己build
自立根生
这个repo的上次build还是2周前
![image](https://github.com/user-attachments/assets/b8ca760e-2876-441e-acae-616608441342)
xiaoyaliu/alist最新的是v.0806.0047(H)

**Reply by yufulings** [reply]
想修改xiaoya镜像为原版，改后metadata后面一直显示alist检测链接

**Reply by jokunhe** [reply]
请教下大佬 ，怎么弄呀，如果不方便教 ，能否提供下您修改后的仓库地址

**Comment by SiloRanger** [comment]
这两天试了下，可以替换alist，安装原版alist，在env文件中修改alist地址为本地地址，就可以打通了。
ALIST_ADDR=http://192.168.x.x:5678/
2345端口如果提示500错误，搜索下就有办法，去原版alist配置文件夹下创建指引文件。
